CMakeLists.txt

```diff
@@ -65,7 +65,15 @@ if(WITH_STL)
 
   # Optimize for speed to reduce the hops
   if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
-    set(CMAKE_CXX_FLAGS_SPEED "/O2")
+    if(CMAKE_BUILD_TYPE MATCHES Debug)
+      # Turn off optimizations for DEBUG
+      set(CMAKE_CXX_FLAGS_SPEED "/Od")
```

Comment:
Sounds like `CMAKE_CXX_FLAGS_SPEED` is just used as a temporary variable? Could we give it another name as the `CMAKE_` prefix may imply it is used by CMake?

Reply:
Yes, I agree and will adjust that.

Reply:
Changed the name of it to `MSVC_CXX_OPT_FLAG`